packages/gatsby/src/schema/infer/index.js

```diff
@@ -57,36 +20,28 @@ const addInferredTypes = ({
   const typeNames = putFileFirst(nodeStore.getTypes())
   const noNodeInterfaceTypes = []
 
+  const nodeTypesToInfer = []
+
   typeNames.forEach(typeName => {
     let typeComposer
-    let inferConfig
     if (schemaComposer.has(typeName)) {
       typeComposer = schemaComposer.getOTC(typeName)
-      inferConfig = getInferConfig(typeComposer)
-      if (inferConfig.infer) {
+      // Infer we have enable infer or if it's "@dontInfer" but we have "addDefaultResolvers: false"
```

Comment:
should it be `addDefaultResolvers: true` in the comment?

Reply:
yes